packages/react-components/react-dialog/e2e/DialogTitle.e2e.tsx

```diff
@@ -19,18 +27,20 @@ describe('DialogTitle', () => {
             <Button>Open dialog</Button>
           </DialogTrigger>
           <DialogSurface>
-            <DialogTitle>Dialog title</DialogTitle>
             <DialogBody>
-              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam exercitationem cumque repellendus eaque
-              est dolor eius expedita nulla ullam? Tenetur reprehenderit aut voluptatum impedit voluptates in natus iure
-              cumque eaque?
+              <DialogTitle>Dialog title</DialogTitle>
```

Comment:
Was it just forgotten to move the title into the body ?

Reply:
No, it was before the introduction of `DialogContent`, so `DialogBody` moved from being a sibling of `DialogTitle` to becoming the parent, and now `DialogContent` should be the sibling.